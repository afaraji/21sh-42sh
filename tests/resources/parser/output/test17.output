Parser: Syntax error near '< or >'
Parser: Syntax error near '< or >'
Parser: Syntax error near '<<'
Parser: Syntax error near '>>'
