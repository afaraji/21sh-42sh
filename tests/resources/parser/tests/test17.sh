#!/bin/sh

#echo "[****** Testing for <ls ******]"
../Minishell -t parser "<ls"

#echo "[****** Testing for ls> ******]"
../Minishell -t parser "ls>"

#echo "[****** Testing for <<ls ******]"
../Minishell -t parser "<<ls"

#echo "[****** Testing for ls>> ******]"
../Minishell -t parser "ls>>"
