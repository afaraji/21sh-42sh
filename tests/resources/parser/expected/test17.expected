Parser say OK
Parser: Syntax error near '\n'
Parser: Syntax error near '<<'
Parser: Syntax error near '\n'
